tasks/create-advisory-internal-request/create-advisory-internal-request.yaml

```diff
@@ -57,8 +57,9 @@ spec:
         # Obtain origin workspace from releasePlanAdmission
         origin=$(jq -rc '.spec.origin' $(workspaces.data.path)/$(params.releasePlanAdmissionPath))
 
-        # Extract the advisory key from the data JSON file
+        # Extract the advisory key and signing configMap name from the data JSON file
         advisoryData=$(jq -c "$(params.jsonKey)" $(workspaces.data.path)/$(params.dataPath))
+        configMapName=$(jq -r '.sign.configMapName' $(workspaces.data.path)/$(params.dataPath))
```

Comment:
`configMapName` will be `null` if the field is not found in the `data.json`. Shouldn't we verify somewhere that it's defined? Or at least add `// ""` to have an empty string if it's not?

Reply:
Updated

Reply:
Didn't know about `-e`, quite clever!